### DISCUSSION TITLE
PS3 Eye open driver - Wut?

### DISCUSSION THREAD

**Discussion by solowingpixy666** [opentrack/opentrack#1368] [General]
Hi all, 

So I've been a user of a really old Opentrack version (just built a new rig/etc) and have reinstalled a new version of OpenTrack but was wondering, what's this 'PS3 Eye open driver' thing that's mentioned? Might just be me but absolutely no idea how to install this driver or at least I've had a brainfart moment every time I've tried to install it, what exactly are we supposed to do with it..? How do I find the libusb-win32 dll to install or..? 

Thanks in advance.

**Comment by sthalik** [comment]
It's in the release notes. You need to download zadig from its project's site.

**Reply by solowingpixy666** [reply]
Alright, thanks for that, had to have someone on another forum of mine guide me as to how to do it, can't believe I'm so stupid. Facing another issue: 

"
Can't Open Camera - PS3 Eye Driver Error:R camera init failed: Access denied (insufficient permissions)
"

"
Startup failure
Error occured while loading tracker PointTracker 1.1
Failed to open camera 'PS3 Eye open driver'
"

The program is being run as admin, driver was installed as admin, swapped USB ports and still no avail, Win 10 is giving perms to any apps to use camera and isn't blocking anything.

Any idea what's going south here? 

**Edit**: Got it working, had to AGAIN reinstall the drivers via Zadig but had to 'list all devices' and select interface 0. Thanks a lot!
